packages/amplication-git-utils/src/providers/bitbucket/bitbucket.service.spec.ts

```diff
@@ -87,7 +87,7 @@ describe("bitbucket.service", () => {
           path: "tests/",
           owner: "mr-bucket",
           repositoryName: "my-repo",
-          repositoryGroupName: "my-group",
+          groupName: "my-group",
```

Comment:
Always prefer consistency:
either `repositoryName` and `repositoryGroupName` or `name` and `groupName`